test_scripts/Polices/build_options/015_ATF_P_TC_Sending_PTS_to_Mobile_Application_HTTP.lua

```diff
@@ -64,18 +64,18 @@ function Test:TestStep_Sending_PTS_to_mobile_application()
 
   EXPECT_HMINOTIFICATION("BasicCommunication.OnAppRegistered", {application = { appName = config.application1.registerAppInterfaceParams.appName } })
   :Do(function()
-    local endpoints_table = commonFunctions:get_data_policy_sql(config.pathToSDL.."/storage/policy.sqlite", "select url from endpoint where service = '0x07' and application_id = 'default'")
-    for _,v in pairs(endpoints_table) do
-      endpoints_url = v
-    end
+      local endpoints_table = commonFunctions:get_data_policy_sql(config.pathToSDL.."/storage/policy.sqlite", "select url from endpoint where service = '0x07' and application_id = 'default'")
+      for _,v in pairs(endpoints_table) do
+        endpoints_url = v
+      end
 
-    local timeout_table = commonFunctions:get_data_policy_sql(config.pathToSDL.."/storage/policy.sqlite", "select timeout_after_x_seconds from module_config")
-    for _,v in pairs(timeout_table) do
-      timeout = v
-    end
+      local timeout_table = commonFunctions:get_data_policy_sql(config.pathToSDL.."/storage/policy.sqlite", "select timeout_after_x_seconds from module_config")
+      for _,v in pairs(timeout_table) do
+        timeout = v
+      end
 
-    EXPECT_NOTIFICATION("OnSystemRequest", {requestType = "HTTP", url = endpoints_url, timeout = timeout})
-  end)
+      EXPECT_NOTIFICATION("OnSystemRequest", {requestType = "LOCK_SCREEN_ICON_URL"}, {requestType = "HTTP", url = endpoints_url, timeout = timeout}): Times(2)
```

Comment:
@istoimenova, 
Since type of the value of _timeout_ is String and type of _timeout_ in API is Integer casting is required. Please correct.